### ISSUE TITLE
Performance tune templating engine

### ISSUE THREAD

**Issue by adelbertc** [getnelson/nelson#117]
Currently our Scalate template engine usage takes a while (couple of seconds) to load + render the template, at least on the very first request (haven't tested with more than one yet) - sometimes this is enough to trigger the 3 second timeout. We should look into this more carefully and fix/tune and problems.

**Comment by timperrett** [comment]
I don't think this is an issue anymore - my prod deployment is pretty snappy when proofing blueprints

**Comment by adelbertc** [comment]
Some initial anecdotal experience suggests the slowdown only happens on the initial load-from-classpath for the default blueprints. For instance the first deployment I did today that used a default blueprint took a couple seconds to load the template from the classpath and then render it, and actually timed out. Subsequent ones finished very fast though.

Maybe we can make it load the default blueprints at startup?
